MoltenVK/MoltenVK/Utility/MVKLogging.h

```diff
@@ -128,7 +128,7 @@ extern "C" {
 #	define MVK_LOG_LEVEL_INFO		MVK_LOGGING_ENABLED
 #endif
 #ifndef MVK_LOG_LEVEL_DEBUG
-#	define MVK_LOG_LEVEL_DEBUG		(MVK_LOGGING_ENABLED && MVK_CONFIG_DEBUG)
+#	define MVK_LOG_LEVEL_DEBUG		MVK_LOGGING_ENABLED
```

Comment:
Did you mean to add this change? This will result in debug spam anytime logging is enabled.

Reply:
Thanks for checking, but yes, it's intentional.

Not so much spam, since the config would have to be set to `MVK_CONFIG_LOG_LEVEL_DEBUG` for it to actually show up in the log. 

What the change to `MVK_LOG_LEVEL_DEBUG` does is remove the conditional compilation of the logging call, so the negative impact will be the overhead of a report/log call that will almost never be seen.

I made the change to normalize debug logging. I thought it would be useful to be able to allow a user in the wild to be able to trigger debug logging, by just adjusting an env var, without having to go through a rebuild of MoltenVK as a debug build.

I'm not 100% sure the convenience/performance trade-off is worth it, but felt it might be helpful for debugging in the wild. I'm happy to hear thoughts and suggestions about that trade-off.

Reply:
That makes sense. It's fine. Hopefully, this will make it easier to diagnose issues in the wild.